Library/Homebrew/cleanup.rb

```diff
@@ -718,11 +718,14 @@ def self.autoremove(dry_run: false)
       # the cache of installed formulae may no longer be valid.
       Formula.clear_cache unless dry_run
 
-      formulae = Formula.installed
+      formulae = Formula.installed(prefer_stub: true)
```

Comment:
Anything dealing with installed formulae should just be able to use the formula from the Cellar rather than a stub?